### PR TITLE
run pre-commit autoupdate

### PR DESCRIPTION
**Pull request by graingert**
Closes #xxxx

- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

File: distributed/diagnostics/progress.py [hunk 1]
```diff
@@ -296,7 +296,7 @@ class AllProgress(SchedulerPlugin):
 
     def __init__(self, scheduler):
         self.all = defaultdict(set)
-        self.nbytes = defaultdict(lambda: 0)
+        self.nbytes = defaultdict(int)
```
Comment (graingert) [inline distributed/diagnostics/progress.py:299]: these are changed due to https://github.com/asottile/pyupgrade/pull/901/files

File: .flake8 [hunk 2]
```diff
@@ -20,6 +20,8 @@ ignore =
     B950
     # No explicit stacklevel in warnings.warn. FIXME we should correct this in the code
     B028
+    # do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
+    E721
```
Comment (hendrikmakait) [inline .flake8:24]: Why ignore this?

**Review by hendrikmakait — APPROVED** [review]
Thanks, @graingert! I have one question that can be dealt with in a follow-up.

File: .flake8 (hunk 2 above)
Comment (graingert) [reply .flake8:24]: We use it all over the place and it's valid code using isinstance changes the behaviour of the code

**Pull request merged**